KeepingYouAwake/Base.lproj/MainMenu.xib

```diff
@@ -681,51 +681,99 @@
+                            </menuItem>
+                            <menuItem title="Set Default: 1 hour" tag="3600" alternate="YES" id="fUA-NX-hxF">
+                                <modifierMask key="keyEquivalentModifierMask" option="YES"/>
+                                <connections>
                                     <action selector="selectTimeInterval:" target="bOj-Ka-Izn" id="5K0-m6-2ol"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="2 hours" tag="7200" id="n3h-ZB-CLV">
+                            <menuItem title="2 hours" tag="7200" id="T5i-64-xaV">
                                 <modifierMask key="keyEquivalentModifierMask"/>
                                 <connections>
+                                    <action selector="selectTimeInterval:" target="bOj-Ka-Izn" id="hFf-G6-uPk"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem title="Set Default: 2 hours" tag="7200" alternate="YES" id="n3h-ZB-CLV">
+                                <modifierMask key="keyEquivalentModifierMask" option="YES"/>
+                                <connections>
                                     <action selector="selectTimeInterval:" target="bOj-Ka-Izn" id="IGV-0t-w8P"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="5 hours" tag="18000" id="tAq-iH-rdw">
+                            <menuItem title="5 hours" tag="18000" id="18N-iz-aod">
                                 <modifierMask key="keyEquivalentModifierMask"/>
                                 <connections>
+                                    <action selector="selectTimeInterval:" target="bOj-Ka-Izn" id="lCt-Xc-iRp"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem title="Set Default: 5 hours" tag="18000" alternate="YES" id="tAq-iH-rdw">
+                                <modifierMask key="keyEquivalentModifierMask" option="YES"/>
+                                <connections>
                                     <action selector="selectTimeInterval:" target="bOj-Ka-Izn" id="bFW-MQ-As3"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="Indefinitely" id="VeD-dm-81B">
+                            <menuItem title="Indefinitely" id="5EU-sH-Q12">
                                 <modifierMask key="keyEquivalentModifierMask"/>
                                 <connections>
+                                    <action selector="selectTimeInterval:" target="bOj-Ka-Izn" id="00X-0X-WzP"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem title="Set Default: Indefinite" alternate="YES" id="VeD-dm-81B">
```

Comment:
The modifier title does not match the default title:

`Set Default: Indefinitely` would be better.